contributor/04-mechanics-of-contributing-article.md

```diff
@@ -69,13 +59,7 @@ Does that imply that in-person communication is forbidden?
 Clearly not: Where written communication shines when it comes to archiving and searchability, in-person communication shines when it comes to communication bandwidth. 
 Try to make time to meet the people behind the names. If possible, try to meet them over your favorite beverage or some food.
 When you're able to hear people speak, when you know their idiosyncrasies remote collaboration will become easier. 
-
-[EDITOR NOTE: There was actually a paper on how trust is build up on remote/asynchronous collaboration. Maybe I'll find that again. 
-The basic point is that aside of reliability in collaboration, continued presence the effect of water cooler discussions and food/drink consumption is quite impressive...
-]  
-[EDITOR NOTE: There was actually a paper on how trust is build up on remote/asynchronous collaboration. Maybe I'll find that again. 
-The basic point is that aside of reliability in collaboration, continued presence the effect of water cooler discussions and food/drink consumption is quite impressive...
-]  
```

Comment:
Was that by intention that you deleted both paragraphs?

Reply:
This is the same note as the one for 44-46, thus removing both. Comment is addressed above.